travis/hibench.conf

```diff
@@ -61,3 +60,6 @@ hibench.dfsioe.dir.name.output	${hibench.workload.dir.name.output}
 
 hibench.workload.dir.name.input         Input
 hibench.workload.dir.name.output        Output
+
+hibench.masters.hostnames  root
+hibench.slaves.hostnames  root
```

Comment:
- What if we don't indicate root here? 
- Style: empty line

Reply:
@gczsjdy  if don't indicate root here ,travis would work fail . 

![image](https://user-images.githubusercontent.com/45281494/89746806-469d3180-daee-11ea-9f6c-2e998ff9e023.png)